### ISSUE TITLE
Reproducible Workflow paper for lesson

### ISSUE THREAD

**Issue by camilavargasp** [NCEAS/learning-hub-organization#67]

**Comment by mbjones** [comment]
@camilavargasp I saw that paper too, very nice. Note that we had a workflows lesson from the last Delta where I reviewed the `targets` workflow package that the paper mentions. This was a very high-level summary, and the main objective of the mini-lesson was to get people more into "workflow thinking", even in their own scripts, which is what the first part of the lesson covered. We developed Kepler, one of the other packages mentioned in the paper, and we learned a lot about what works and what doesn't with workflows in that foray deep into workflows. And much of that thinking informed our lessons on `parsl` and `ray` in the python course, which are much more mature than `targets` in R.  The lesson could use more work, and this space changes rapidly. Here's the lesson link: https://learning.nceas.ucsb.edu/2021-11-delta/session-6-reproducible-workflows-revisited.html
